sample/front/index.html

```diff
@@ -39,7 +39,7 @@ <h2>Actions</h2>
       <button id="reset" disabled>Reset</button>
     </div>
 
-    <script src="http://localhost:3005/main.js"></script>
+    <script src="http://localhost:3005/index.js"></script>
```

Comment:
Seems like this a breaking change. Can we still use `main.js` or do we need `index.js` for any reason?

Reply:
Cannot be changed